### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by RIMPA-SEN**

### CODE REVIEW

File: src/main/java/org/example/Main.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package org.example;
+
+public class Main {
+    public static void main(String[] args) {
+        System.out.println("Hello world!");
+        System.out.println();
```
Comment (AnilPatamber) [inline src/main/java/org/example/Main.java:6]: What is the use of this line?

Comment (AnilPatamber) [reply src/main/java/org/example/Main.java:6]: remove it if not required

**Review by AnilPatamber — CHANGES_REQUESTED** [review]

File: src/main/java/org/example/Main.java (hunk 1 above)
Comment (RIMPA-SEN) [reply src/main/java/org/example/Main.java:6]: Okay will remove that

**Review by AnilPatamber — APPROVED** [review]
great work

**Pull request merged**